app/plugins/eu.esdihumboldt.hale.app.transform/src/eu/esdihumboldt/hale/app/transform/ExecApplication.groovy

```diff
@@ -150,6 +153,8 @@ $baseCommand
   control which files to load.
   If you do not specify -include, it defaults to "**", i.e. all files being
   included, even if they are in sub-directories.
+  If you specify more than one filter ( e.g. -filter or -filter-on) will be applied as 'OR' on each instance. 
```

Comment:
You inserted that in the middle of the section explaining the usage of directories as sources. Please create a new paragraph with your content. Maybe add a small intro that filters are applied to source data, before stating what happens with multiple filters.